.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "singleQuote": true,
+  "semi": true,
+  "bracketSpacing": true,
+  "trailingComma": "none",
```

Comment:
what does trailing comma?

Reply:
https://codeburst.io/learn-javascript-es-2017-trailing-commas-4534891f3f90

Reply:
Adding coma after last element of array or object. I don;t like it :p

Reply:
me too